src/Pages/ViewList.js

```diff
@@ -1,9 +1,26 @@
 import React from 'react';
+import { useCollection } from 'react-firebase-hooks/firestore';
+import { db } from '../lib/firebase';
 
-const ViewList = () => (
-  <div>
-    <h1 className="title is-1">View List</h1>
-  </div>
-);
+const ViewList = () => {
+  const [value, loading, error] = useCollection(db.collection('lists'));
```

Comment:
Eventually we'll pull in the `token` as a prop to `ViewList` and put it here:
```suggestion
  const [value, loading, error] = useCollection(db.collection(token));
```